tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/demo/tf-interactive-inference-age-demo.html

```diff
@@ -123,14 +123,24 @@
             const indices = Object.keys(this.indicesToInfer).sort();
             inferences.indices = indices;
             inferences.results = [{regressionResult: {regressions: []}}];
-            const tensors = [];
+            let tensors = null;
```

Comment:
Correct me if I am wrong but it seems easier to parse and read to create an array and use `tf.concat` like this:
```ts
const tensorArr = indices.map(idx => this.convertExToTensor(this.data[idx]));
const tensors = tf.concat(tensorArr);
// Does concatting create a new tensor? Is it okay to dispose it here?
tensorArr.forEach(tensor => tensor.dispose());
const input = tensors.reshape([indices.length, 105]);
```

Reply:
good suggestion. changing.